client/src/app/pages/applications/applications-table-analyze/applications-table-analyze.tsx

```diff
@@ -346,7 +346,9 @@ export const ApplicationsTableAnalyze: React.FC = () => {
         },
         {
           title: t("actions.delete"),
-          isDisabled: row.migrationWave !== null,
+          isAriaDisabled: row.migrationWave !== null,
+          tooltip: "Cannot delete application assigned to a migration wave.",
+          tooltipProps: "position: 'top'",
```

Comment:
@gildub the PF type `any` for `tooltipProps` isn't helping here, but looking internally it expects an object here and not a string.

```ts
tooltipProps: { position: "top" },
```